gno.land/pkg/keyscli/run.go

```diff
@@ -98,16 +99,25 @@ func execMakeRun(cfg *MakeRunCfg, args []string, cmdio commands.IO) error {
 			if err != nil {
 				return fmt.Errorf("could not read %q: %w", sourcePath, err)
 			}
-			memPkg.Files = []*std.MemFile{
-				{
-					Name: info.Name(),
-					Body: string(b),
+
+			gm := gno.ParseMemMod(filepath.Dir(sourcePath))
+			// TODO(hariom): Remove
+			// if err != nil {
+			// 	return fmt.Errorf("error parse gno.mod at: %q", sourcePath)
+			// }
```

Comment:
leftover?

Reply:
yeah. I will remove it in the cleanup once I fix all the failing tests.